peloader/winapi/Strings.c

```diff
@@ -86,12 +86,24 @@ STATIC int WINAPI WideCharToMultiByte(UINT CodePage, DWORD dwFlags, PVOID lpWide
         strcpy(lpMultiByteStr, ansi);
         free(ansi);
         return strlen(lpMultiByteStr) + 1;
+    } else if (!lpMultiByteStr && cbMultiByte == 0) {
+        return strlen(ansi) + 1;
```

Comment:
I think ansi needs to be free()d here.

Reply:
Good catch, you're right. My C's a little rusty.